modules/sdk-unified-wallet/test/unit/ecdsaUnifiedWallets.ts

```diff
@@ -125,7 +125,7 @@ describe('EVM Wallets:', function () {
 
     it('should get all unified wallets', async function () {
       nock(bgUrl).get('/api/v2/wallet/evm').reply(200, [expected]);
-      const result = await evmWallets.getAllUnifiedWallets();
+      const result = await evmWallets.getUnifiedWallets();
```

Comment:
Pagination was[ unit tested](https://github.com/BitGo/bitgo-microservices/blob/904ca39df91bbe2f8048dc9a8b39380734f40c6d/packages/wallet-platform/test/api/v2/EvmWallet/EvmWallet.ts#L363) in `wp`, since this is an API call we could test in an integration test.